src/ripple/site_scons/scons_to_ninja.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2014 The Native Client Authors. All rights reserved.
```

Comment:
@rec The formatting and the local functions make this file very non-pythonic. I agree. I may re-write it some weekend, but for now I'd rather just import it whole from the chromium project.